src/serializer/ResidualFunctions.js

```diff
@@ -117,6 +117,51 @@ export class ResidualFunctions {
     if (!this.firstFunctionUsages.has(val)) this.firstFunctionUsages.set(val, bodyReference);
   }
 
+  _shouldUseFactoryFunction(funcBody: BabelNodeBlockStatement, instances: Array<FunctionInstance>) {
+    function shouldInlineFunction(): boolean {
+      let shouldInline = !funcBody;
```

Comment:
It is not clear to me what you are doing here. According to the type signature funcBody is never null/undefined, hence !funcBody is equivalent to false.

Reply:
I copied from existing code. Seems out-dated code, will fix.